### DISCUSSION TITLE
Previous integration data lost?

### DISCUSSION THREAD

**Discussion by nigelspowage** [nathanmarlor/foxess_modbus#222] [Q&A]
I recently switched my Home Assistant integration from StealthChesnut to NathanMarlor's. The historical data from the old integration isn't showing in the Energy tab on HA. Is there a way to get this displayed ? Have I done something wrong in the install ?

**Comment by canton7** [comment]
I've been told that following https://github.com/nathanmarlor/foxess_modbus/wiki/Migrating-from-HA-FoxESS-Modbus means that your sensors get transfered over, so they'll keep their history

**Reply by nigelspowage** [reply]
Unfortunately this doesn't appear to have happened to me.

**Reply by nigelspowage** [reply]
Before a couple of days ago I had data from November last year.  
This is what my energy dashboard looks like now. :(
![image](https://user-images.githubusercontent.com/34656171/236545634-8fe81a23-1e50-492a-8fb8-eefa3e2ef49a.png)

**Comment by nigelspowage** [comment]
I've had a thought of what may have happened.

The instructions to remove the customer sensors added by stealthchesnut for battery charge, battery discharge and  grid consumption. I assumed this meant to also remove the other entries for 'utility meter'.

So I had no utility meter entries when I installed this new integration.

I've re-added the utility meter entries but my data has not re-appeared. 

Is this the cause of my historical data being removed ? if so is it gone forever or is there a way to get it back ?
